src/test/java/io/sniffy/junit/QueryCounterTest.java

```diff
@@ -21,18 +22,30 @@ public void testNotAllowedQueriesByDefault() {
     }
 
     @Test
-    @Expectation(1)
+    @Expectation(count = @Count(1))
     public void testAllowedOneQuery() {
         executeStatement();
     }
 
     @Test
-    @Expectation(value = 5, atLeast = 2)
+    @Expectation(1)
+    public void testAllowedOneQuery_DeprecatedApi() {
+        executeStatement();
+    }
+
+    @Test
+    @Expectation(count = @Count(value = 5, min = 2))
     public void testAmbiguousExpectationAnnotation() {
         thrown.expect(IllegalArgumentException.class);
     }
 
     @Test
+    @Expectation(value = 5, atLeast = 2)
+    public void testAmbiguousExpectationAnnotation_DeprecatedApi() {
+        thrown.expect(IllegalArgumentException.class);
+    }
+
+    @Test
     @Expectation(2)
     @NoQueriesAllowed
     public void testAmbiguousAnnotations() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/sniffy/sniffy/file/2495677739/issues/source?bid=3005960&fileBranchId=3034483#l51)